wave_geometry/CMakeLists.txt

```diff
@@ -1,22 +1,26 @@
 PROJECT(wave_geometry)
 
 # EXTERNAL PACKAGES
-FIND_PACKAGE(kindr)
+FIND_PACKAGE(kindr REQUIRED)
```

Comment:
should Kindr be capitalized here?

Reply:
No, it must match the name of `kindrConfig.cmake`